integrations/viborg/viborg.py

```diff
@@ -1,36 +1,39 @@
-import os
 import csv
 import json
 import pathlib
 import datetime
 from chardet.universaldetector import UniversalDetector
 
+from integrations import cpr_mapper
 from os2mo_data_import import ImportHelper
-from integrations.SD_Lon.sd_importer import  sd_importer
+from integrations.SD_Lon import sd_importer
 from integrations.ad_integration import ad_reader
 
-cfg_file = pathlib.Path.cwd() / 'settings' / 'kommune-viborg.json'
+# TODO: Soon we have done this 4 times. Should we make a small settings
+# importer, that will also handle datatype for specicic keys?
+cfg_file = pathlib.Path.cwd() / 'settings' / 'settings.json'
 if not cfg_file.is_file():
     raise Exception('No setting file')
 settings = json.loads(cfg_file.read_text())
 
-MANAGER_FILE = os.environ.get('MANAGER_FILE')
-GLOBAL_GET_DATE = datetime.datetime(2019, 9, 15, 0, 0)
+cpr_map = pathlib.Path.cwd() / 'settings' / 'cpr_uuid_map.csv'
```

Comment:
Den ville være fed at få parameteriseret - altså ingen stress - bare derhenad - af samme grund - så kan vi læse settingsfilen for at vide, hvad der skal bakkes op.